kii_thing_if.h

```diff
@@ -106,10 +106,24 @@ typedef kii_bool_t
          KII_THING_IF_WRITER writer);
 
 /**
+ * callback function for handling notification.
+ * @param [in] kii_t kii_t object if you want to send request to kii
+ * cloud you can use this kii_t object.
+ * @param [in] message message from pushing to application.
```

Comment:
> Can you improve message description ?

Instead of improving message description, I added explanation of this handler. What do you think?